tests/perf/perf_tests_perf.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "seastar/core/future.hh"
+#include "seastar/coroutine/maybe_yield.hh"
+#include "seastar/util/later.hh"
+#include <seastar/core/coroutine.hh>
+#include <seastar/testing/perf_tests.hh>
```

Comment:
\<> includes please.

Reply:
@tchaikov could we enforce it? By using `-isystem include` instead of `-Iinclude`, though it would take me several weeks to figure out how to do it with cmake.

Reply:
Actually it wouldn't help.

Reply:
sorry this is how clangd auto-adds headers, let me see if I can see how to do this right for the seastar way

Reply:
fixed in push

Reply:
> @tchaikov could we enforce it? By using `-isystem include` instead of `-Iinclude`, though it would take me several weeks to figure out how to do it with cmake.

created #2592

Reply:
Agree that not reporting errors would make it worse.

Reply:
After poking the LLVM brear it looks like this _will_ be available in a future clangd version: https://github.com/llvm/llvm-project/pull/67749#issuecomment-2563964013